wp-content/themes/core/assets/css/src/theme/typography/lists.pcss

```diff
@@ -65,6 +66,7 @@ li {}
 			left: 0;
 			font-size: var(--font-size-body-large);
 			line-height: 2rem; /* list item line-height */
+			color: var(--color-primary);
```

Comment:
I noticed we are applying the primary color for stylized `ol` lists. Should that be the case here as well?

Reply:
I don't think so, no. That's to provide the colored numbers for the stylized OL, per the [Figma file](https://www.figma.com/file/Wj2g7HVyBQWYIuxgNKMCxf/SQ1-Main-Patterns?node-id=5069%3A292).